src/ostorlab/runtimes/local/app/app.py

```diff
@@ -0,0 +1,13 @@
+import flask
+from graphene_file_upload import flask as graphene_upload_flask
+
+from ostorlab.runtimes.local.app.schema import schema
+
+flask_app = flask.Flask(__name__)
+
+flask_app.add_url_rule(
+    "/oxo_graphql",
```

Comment:
```suggestion
    "/graphql/",
```

Reply:
maybe this should be oxo for consistancy?